src/config.rs

```diff
@@ -481,6 +483,10 @@ fn expand_directory(source: &Path, target: FileTarget) -> Result<Files> {
         let mut map = Files::new();
         map.insert(source.into(), target);
         Ok(map)
+    } else if let FileTarget::Symbolic(SymbolicTarget{target:_, owner:_, condition: _, recursive: Some(false)}) = target {
```

Comment:
This can be `if let FileTarget::Symbolic(SymbolicTarget { recursive: false, .. })` with the previous change